### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by jacekelgda** [jacekelgda/mr.moody#27]
We need more tests. Let's add some. I will link it with CI.

**Comment by gabrielsclimaco** [comment]
Are there some thing already set up?

**Comment by jacekelgda** [comment]
Yes Sir there is. Just a single unit test. https://github.com/x-team/mr.moody/blob/master/modules/__tests__/attachmentsResolver.test.js

We are testing with `jest`.
Now, I think there is little need for unit tests in here. It might be cool to add some mocks for services we use and write tests for some of the modules ( chat, report ). Possibly controllers as well.

Chat is using slack api and report firebase api.
